Error in building spike!

Hello, I cloned the repository I built the LLVM toolchain no problembt when I attempt to build to bild Spike i Get this error:

**../riscv/interactive.cc: In member function ‘void sim_t::interactive_fregd_all(const string&, const std::vector<std::__cxx11::basic_string<char> >&)’:
../riscv/interactive.cc:639:21: error: ‘class processor_t’ has no member named ‘get_max_xlen’; did you mean ‘get_xlen’?
  639 |   int max_xlen = p->get_max_xlen();
      |                     ^~~~~~~~~~~~
      |                     get_xlen**

How can I fix this?
Thank you.

--- a/patches/0003-riscv-isa-sim-patch
+++ b/patches/0003-riscv-isa-sim-patch
@@ -71,7 +71,7 @@ index ffc706bd..95b391a6 100644
 +    throw trap_interactive();
 +
 +  processor_t *p = get_core(args[0]);
-+  int max_xlen = p->get_max_xlen();
++  int max_xlen = p->get_isa().get_max_xlen();
 +
 +  std::ostream out(sout_.rdbuf());
 +  out << std::hex;

